tests/hipscat_import/catalog/test_run_round_trip.py

```diff
@@ -411,3 +411,51 @@ def test_import_hipscat_index(
         data_frame.columns,
         ["id", "Norder", "Dir", "Npix"],
     )
+
+
+@pytest.mark.dask
+def test_import_gaia_minimum(
+    dask_client,
+    formats_dir,
+    tmp_path,
+):
+    """Test end-to-end import, using a representative chunk of gaia data."""
+    input_file = os.path.join(formats_dir, "gaia_minimum.csv")
+    schema_file = os.path.join(formats_dir, "gaia_minimum_schema.parquet")
+
+    args = ImportArguments(
+        output_artifact_name="gaia_minimum",
+        input_file_list=[input_file],
+        input_format="csv.gz",
+        file_reader=CsvReader(
+            comment="#",
+            schema_file=schema_file,
```

Comment:
I've got an error when I was trying to run the same code with file `GaiaSource_310878-313367.csv.gz`. However adding `parquet_kwargs={"dtype_backend": "numpy_nullable"}` helped me to read the file